Directory.Packages.props

```diff
@@ -12,7 +12,7 @@
     <PackageVersion Include="BenchmarkDotNet.Annotations" Version="0.12.1" />
     <PackageVersion Include="BenchmarkDotNet" Version="0.12.1" />
     <PackageVersion Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildVersion)" />
-    <PackageVersion Include="Microsoft.Build.Locator" Version="1.2.6" />
+    <PackageVersion Include="Microsoft.Build.Locator" Version="1.4.1" />
```

Comment:
This verison of MSBuildLocator includes loading dependencies from the MSBuild path.